src/components/InventoryGroups/utils/api.js

```diff
@@ -25,6 +25,13 @@ export const validateGroupName = (name) => {
     .then((resp) => resp?.results.some((group) => group.name === name));
 };
 
+export const updateGroupById = (id, payload) => {
+    return instance.put(`${INVENTORY_API_BASE}/groups/${id}`, {
+        Name: payload.name,
```

Comment:
according to openapi there is only `name` field:
```suggestion
        name: payload.name,
```

Reply:
Thanks, fixed